proto/regen/ecocredit/marketplace/v1beta1/state.proto

```diff
@@ -4,55 +4,58 @@ package regen.ecocredit.marketplace.v1beta1;
 
 import "google/protobuf/timestamp.proto";
 import "cosmos/orm/v1alpha1/orm.proto";
-import "cosmos/base/v1beta1/coin.proto";
 import "regen/ecocredit/marketplace/v1beta1/types.proto";
 
 // SellOrder represents the information for a sell order.
 message SellOrder {
     option (cosmos.orm.v1alpha1.table) = {
         id: 1,
-        primary_key: {fields: "order_id"}
-        index: {id: 1 fields: "batch_denom"}
-        index: {id: 2 fields: "owner"}
+        primary_key: {fields: "id", auto_increment: true}
+        index: {id: 1 fields: "batch_id"}
+        index: {id: 2 fields: "seller"}
         index: {id: 3 fields: "expiration"}
     };
 
-    // order_id is the unique ID of sell order.
-    uint64 order_id = 1;
+    // id is the unique ID of sell order.
+    uint64 id = 1;
 
-    // owner is the address of the owner of the credits being sold.
-    string owner = 2;
+    // seller is the address of the owner of the credits being sold.
+    bytes seller = 2;
```

Comment:
Should it be a string
```suggestion
    string seller = 2;
```

Reply:
in the state tables I think we actually want to save space by using a bytes account address. in Msg's we want to usestring